src/hal/drivers/mesa-hostmot2/hm2_rpspi.c

```diff
@@ -0,0 +1,568 @@
+/*    This is a component for RaspberryPi to hostmot2 over SPI for linuxcnc.
+ *    Copyright 2016 Matsche <tinker@play-pla.net>
+ *
+ *    This program is free software; you can redistribute it and/or modify
+ *    it under the terms of the GNU General Public License as published by
+ *    the Free Software Foundation; either version 2 of the License, or
+ *    (at your option) any later version.
+ *
+ *    This program is distributed in the hope that it will be useful,
+ *    but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *    GNU General Public License for more details.
+ *
+ *    You should have received a copy of the GNU General Public License
+ *    along with this program; if not, write to the Free Software
+ *    Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ */
+
+/* Without Source Tree */
+#undef WOST
```

Comment:
WOST is a acronym for "Without Source Tree". It's a switch and helps building the driver with halcompile.
i.e.: https://github.com/tinkercnc/spi-fpga-driver/tree/master/RaspberryPi/sandbox